conf/logs.json

```diff
@@ -308,5 +308,524 @@
         }
       }
     }
-  }
-}
\ No newline at end of file
+  },
+  "bro:conn.log": {
+    "parser": "json",
+    "schema": {
+      "ts": "string",
+      "uid": "string",
+      "id": "string",
+      "proto": "string",
+      "service": "string",
+      "duration": "string",
+      "orig_bytes": "integer",
+      "resp_bytes": "integer",
+      "conn_state": "string",
+      "local_orig": "boolean",
+      "local_resp": "boolean",
+      "missed_bytes": "integer",
+      "history": "string",
+      "orig_pkts": "integer",
+      "resp_ip_bytes": "integer",
+      "tunnel_parents": "string",
+      "orig_l2_addr": "string",
+      "resp_l2_addr": "string",
+      "vlan": "integer",
+      "inner_vlan": "integer"
+    }
+  },
+  "bro:dce_rpc.log": {
+    "parser": "json",
+    "schema": {
+      "ts": "string",
+      "uid": "string",
+      "id": "string",
+      "rtt": "string",
+      "named_pipe": "string",
+      "endpoint": "string",
+      "operation": "string"
+    }
+  },
+  "bro:dhcp.log": {
+    "parser": "json",
+    "schema": {
+      "ts": "string",
+      "uid": "string",
+      "id": "string",
+      "mac": "string",
+      "assigned_ip": "string",
+      "lease_time": "integer",
+      "trans_id": "integer"
+    }
+  },
+  "bro:dnp3.log": {
+    "parser": "json",
+    "schema": {
+      "ts": "string",
+      "uid": "string",
+      "id": "string",
+      "fc_request": "string",
+      "fc_reply": "string",
+      "iin": "integer"
+    }
+  },
+    "bro:dns.log": {
+    "parser": "json",
+    "schema": {
+      "ts": "string",
+      "uid": "string",
+      "id": "string",
+      "proto": "string",
+      "trans_id": "string",
+      "rtt": "string",
+      "query", "string",
```

Comment:
`"query": "string",`

Please lint the file to catch other errors like this.  Thanks!